src/main/java/pages/AppSettingsPage.java

```diff
@@ -75,7 +78,8 @@ public AppSettingsPage switchToSpanish() {
         spanishLanguage.shouldBe(Condition.visible).click();
         if (isIOS()) crossBtn.shouldBe(Condition.visible).click();
         cryptoAccountBtnSpanish.shouldBe(Condition.visible);
-        if (isAndroid()) buyXorWithFiatBtnSpanish.shouldBe(Condition.visible);
+        //if (isAndroid()) buyXorWithFiatBtnSpanish.shouldBe(Condition.visible);
```

Comment:
temporarily left this check under the comment, I’m waiting for clarification on this button.